src/scss/form.scss

```diff
@@ -45,6 +45,10 @@
       }
    }
 
+   select:not([multiple]) {
+      height: calc(#{$spacing-xx-small + $spacing-xx-small + $line-height-small} + 2px);
```

Comment:
What is the `2px` is there a variable that you could use instead? Or is it just a *"random"* number?